icons/flag-off.svg

```diff
@@ -11,5 +11,5 @@
 >
   <path d="M8 2c3 0 5 2 8 2s4-1 4-1v11"/>
   <path d="M4 22v-7m0 0s1-1 4-1 5 2 8 2M4 15V4"/>
-  <path d="m2 2 20 20"
+  <path d="m2 2 20 20"/>
```

Comment:
This is my bad, also, we should probably use a `<line>` element as other `*-off` icons do instead of a `<path>`.
```suggestion
  <line x1="2" y1="2" x2="22" y2="22" />
```

Reply:
Changed to line